deploy/test-app/apply/TestAppGateway.yaml

```diff
@@ -78,4 +78,14 @@ spec:
       get:
         x-fabric-employee-access:
           user-list:
-            - notreal
\ No newline at end of file
+            - notreal
+    /static:
+      get:
+        x-fabric-ratelimits:
+          default-rate: 2
+        x-fabric-privileges:
+          - fabric-demo-app.read
+        x-fabric-static-response:
+          status: 503
+          content-type: application/problem+json
```

Comment:
Does it make sense to include support for headers as well?

Reply:
Like any Map<String, String>? It's the most generic approach yeah. That's probably better, can switch to that.